mne/io/base.py

```diff
@@ -658,7 +658,7 @@ def filenames(self):
         return tuple(self._filenames)
 
     @annotations.setter
-    def annotations(self, annotations):
+    def annotations(self, annotations, emit_warning=True):
```

Comment:
@jaeilepp can you tell me where we have a param to decide if warning should be triggered or not?